net/mwan3/Makefile

```diff
@@ -20,6 +20,7 @@ define Package/mwan3
    CATEGORY:=Network
    SUBMENU:=Routing and Redirection
    DEPENDS:= \
+     +@BUSYBOX_CUSTOM +@BUSYBOX_CONFIG_DIFF +busybox \
```

Comment:
Won't this add diff to busybox at compile time? It will use extra bytes for everyone, even those not using mwan3